src/index.js

```diff
@@ -90,6 +135,17 @@ const extendCodeBlockWithSyntaxHighlighting = ( settings ) => {
 						placeholder={ __( 'Write code…', 'syntax-highlighting-code-block' ) }
 						aria-label={ __( 'Code', 'syntax-highlighting-code-block' ) }
 					/>
+					<div aria-hidden={ true } className="code-block-bg">
```

Comment:
I like this idea of having a `div` appearing behind the textarea. It isn't styled properly for me when testing, but it seems like a good approach.